api/types/constants.go

```diff
@@ -586,6 +608,63 @@ const (
 	TeleportAzureMSIEndpoint = "azure-msi." + TeleportNamespace
 )
 
+// Labels added by the discovery service to discovered databases,
+// Kubernetes clusters, and Windows desktops.
+const (
+	// DiscoveryLabelRegion identifies a discovered cloud resource's region.
+	DiscoveryLabelRegion = "region"
+	// DiscoveryLabelAccountID is the label key containing AWS account ID.
+	DiscoveryLabelAccountID = "account-id"
```

Comment:
Nit: I think it would be good practice to start documenting _why_ a label omits the `TeleportNamespace` prefix if it does, since namespacing should be the assumed default unless some particular reason precludes it.

Reply:
will add to the godoc for other labels before merging, it's only really explained on the azure database name override label

Reply:
oh, actually a lot of these labels aren't namespaced as a matter of legacy. I don't think there is any real reason why we can't or shouldn't use namespaced labels in the Teleport resources. I think we should consider deprecating these and switching to namespaced labels for consistency, but I'd prefer we do that in another PR. I don't know why they weren't namespaced before, but I'll see if anyone else knows.